[BUG] Running Install Guide

When cloning and following the instructions in the README and error occurs on `docker-compose up`

```bash
.
.
.
Step 6/6 : RUN npm install --production
 ---> Running in b1d9996c818e
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t https://github.com/debris/bignumber.js.git
npm ERR! 
npm ERR! fatal: not a git repository: /app/dex-contracts/../.git/modules/dex-contracts
npm ERR! 
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-06-15T06_36_31_174Z-debug.log
ERROR: Service 'truffle' failed to build: The command '/bin/sh -c npm install --production' returned a non-zero code: 1
```

It appears to be an issue with `bignumber` node module... either this hasn't been included in the truffle configuration or needs to be handled separately. I believe @josojo had this same issue yesterday!

--- a/README.md
+++ b/README.md
@@ -44,8 +44,10 @@ Clone the repository, its submodule, and run the container
 ```bash
 git clone git@github.com:gnosis/dex-services.git
 cd dex-services
-git submodule init
-git submodule update
+git submodule update --init
+cd dex-contracts 
+npm install && truffle compile 
+cd ../
 docker-compose up
 ```
 

